ngen_files/rc_files/realization_config_topoflow-glacier_cat3062920.json

```diff
@@ -0,0 +1,30 @@
+{
+    "global": {
+      "formulations":
+      [
+        {
+          "name": "bmi_python",
+          "params": {
+              "python_type": "topoflow_glacier.bmi.bmi_topoflow_glacier.BmiTopoflowGlacier",
+              "model_type_name": "BmiTopoflowGlacier",
+              "init_config": "./data/topoflow-glacier/config/{{id}}.yaml",
+              "main_output_variable": "channel_water_x-section__volume_flow_rate",
+              "uses_forcing_file": false,
+              "variables_names_map" : {
+                  "streamflow_cms": "channel_water_x-section__volume_flow_rate",
+                  "atmosphere_water__precipitation_leq-volume_flux":"atmosphere_water__liquid_equivalent_precipitation_rate" }
+        }
+      }
+      ],
+        "forcing": {
+            "file_pattern": "{{id}}.csv",
+            "path": "./data/topoflow-glacier/forcing/",
+            "provider": "CsvPerFeature"
+        }
+    },
+    "time": {
+        "start_time": "2013-05-01 00:00:00",
+        "end_time": "2013-05-31 23:00:00",
+        "output_interval": 3600
```

Comment:
time steps should be hourly. Like one simulation per hour, not 3600, as far as I remember. so I guess it should be changed to 1.